DigitalExcellence/dex-frontend
# Bugfix/458-edit-and-delete-button-always-shown

## Description

It looks like this problem was caused by some failed merge conflict. The correct code was there but was blocked by the old code.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to the project overview page
2. Click a project
3. Check if there is a cog icon in the top right

## Link to issue

Closes: #458 
Closes: #445
